docs/http/traffic-policy/expressions/variables.mdx

```diff
@@ -88,15 +88,15 @@ Timestamp when the connection to ngrok was started.
 
 The following variables are available under the `conn.geo` namespace:
 
-| Name                                             | Type   | Description                                                                                                   |
-| ------------------------------------------------ | ------ | ------------------------------------------------------------------------------------------------------------- |
-| [`conn.geo.city`](#conngeo_city)                 | string | The name of the city, in EN, where the `conn.client_ip` is likely to originate.                               |
-| [`conn.geo.country`](#conngeo_country)           | string | The name of the country, in EN, where the `conn.client_ip` is likely to originate.                            |
-| [`conn.geo.country_code`](#conngeo_country_code) | string | The two-letter ISO country code where the `conn.client_ip` is likely to originate.                            |
-| [`conn.geo.latitude`](#conngeo_latitude)         | string | The approximate latitude where the `conn.client_ip` is likely to originate.                                   |
-| [`conn.geo.longitude`](#conngeo_longitude)       | string | The approximate longitude where the `conn.client_ip` is likely to originate.                                  |
-| [`conn.geo.radius`](#conngeo_radius)             | string | The radius in kilometers around the latitude and longitude where the `conn.client_ip` is likely to originate. |
-| [`conn.geo.subdivision`](#conngeo_subdivision)   | string | The name of the subdivision, in EN, where the `conn.client_ip` is likely to originate.                        |
+| Name                                            | Type   | Description                                                                                                   |
+| ----------------------------------------------- | ------ | ------------------------------------------------------------------------------------------------------------- |
+| [`conn.geo.city`](#conngeocity)                 | string | The name of the city, in EN, where the `conn.client_ip` is likely to originate.                               |
```

Comment:
will need to update these for the tcp / tls variable tables too since `conn.geo` is available for those types of endpoints too, there's no template/component being used across all of them yet

Reply:
Not sure what ya mean. I tested all links and they work. 🤔

Reply:
good catch! didn't know we had more haha, thank you

Reply:
donezo bunzo!